nbgrader/docs/source/index.rst

```diff
@@ -56,5 +56,5 @@ For an overview and demonstration of nbgrader's core functionality, check out th
     :caption: Resources
 
     Source repository <https://github.com/jupyter/nbgrader>
-    Jupyter education mailing list <http://tinyurl.com/jupyter-education>
```

Comment:
Do you think it's possible to add some exclusion marks and keep the `tinyurl` version? I guess it is there for a reason like to see the number of people going to the Google Groups from this exact link?

Reply:
Noob question 😅 How do I exclude it? Do you mean should I keep tinyurl link? or is there a way to do that at travis config?

Reply:
@npenzin ?

Reply:
@swarnava Soorry for late reply. I'm afraid I can't help you on that but I guess TravisCI docs might have an answer on that, check it out. Best! :)

Reply:
I believe the reason this uses tinyurl is because the `#` in the google groups URL was causing problems with the linkchecker. This might not be a problem anymore, in which case it would be fine to put the full URL.

Reply:
Does this mean we are good to merge this PR now ? 😄

Reply:
Yep I just tested it locally and it seems to be ok. The tests are failing for other reasons which I'll try to investigate separately.